Src/VsInterfaces/ISharedService.cs

```diff
@@ -1,6 +1,8 @@
 ﻿using Microsoft.FSharp.Core;
 using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
+using System.Collections.Generic;

```

Comment:
These changes seem unnecessary here.

Reply:
Sorry, this is left over from when I had the whole thing working through a circuitous path:

- VimCore
- VimHost
- SharedService
- MultiSelectionBroker

That path got much simplified, and I deleted the code, but the usings remained.